Directory.Packages.props

```diff
@@ -3,13 +3,15 @@
     <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.0" />
     <PackageVersion Include="AWSSDK.SQS" Version="3.7.0" />
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="1.1.0" Condition=" '$(TargetFramework)' == 'net461' " />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" Condition=" '$(TargetFramework)' == 'netstandard2.0' " />
     <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageVersion Include="StructureMap" Version="4.6.0" />
     <PackageVersion Include="System.Text.Json" Version="4.6.0" />
     <PackageVersion Include="System.Threading.Channels" Version="4.5.0" />
+    <PackageVersion Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.300" />
```

Comment:
Please add both packages to [dependabot.yml](https://github.com/justeattakeaway/JustSaying/blob/main/.github/dependabot.yml) as an ignored dependency so we "pin" the versions the library depends on.